pkg/cmd/template/schema_test.go

```diff
@@ -2533,6 +2743,141 @@ foo: 0
 
 		assertFails(t, filesToProcess, expectedErr, opts)
 	})
+	t.Run("when schema/default annotation", func(t *testing.T) {
+		t.Run("value is empty", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/default
+foo: 0
+`
+			templateYAML := `#@ load("@ytt:data", "data")
+---
+foo: #@ data.values.foo
+`
+			expectedErr := `
+Invalid schema
+==============
+expected @schema/default annotation to contain default value
+
+schema.yml:
+    |
+  4 | foo: 0
+    |
+
+    = found: missing value
+    = expected: integer (by schema.yml:4)
+    = hint: value must be the same type as the annotated node.
+    = hint: value must be in Starlark format, e.g.: {'key': 'value'}, True.
```

Comment:
These hints _are_ true, but I wonder if they are helpful prompts for the person who (apparently?) forgot to include a value in the annotation.

 ```suggestion
syntax error in @schema/default annotation

schema.yml:
    |
  4 | foo: 0
    |

    = found: missing value (in @schema/default above this item)
    = expected: integer (by schema.yml:4)
```

(also, seeing that we have multiple kinds of errors like this, trying out what it might look like to have a more generic header for errors like this)